### ISSUE TITLE
Update "Docker Image Selection" section of docs when new images are added

### ISSUE THREAD

**Issue by mat-adamec** [CoffeaTeam/coffea-casa#295]
The "Docker Image Selection" section under the "First Steps" document has an outdated picture. Both coffea-casa instances at UNL currently only have one image, so the whole section is thus presently redundant.

For the moment, keeping the outdated picture seems best and serves two purposes: 
1. it shows that other images are possible, which is a nice feature
2. it serves as a placeholder and template for when future images are added

When the second point is addressed, the picture should be updated to include the new images (rather than the outdated ones), and the use case of each image should be described (at present, the section only tells users to select the default one).

In short, there's presently no issue and the "outdatedness" is known/accepted, but one will arise when new images are added and the documentation should be updated accordingly then.
